250-bug-link-of-operational-epds-leads-back-to-main-site

baho-j opened the pull request:
# Fix Generic Operational EPDs Link Display

  Closes #250

# What this does

Fixes a bug where generic operational EPDs showed a clickable link that incorrectly redirected to the home page instead of displaying a "Generic" pill badge like in the EPD search functionality.

## Main features
  - Conditional Link Display: Only show external source links for official EPDs
  - Generic EPD Badge: Display "Generic" pill for generic EPDs instead of broken link
  - Consistent UI Pattern: Matches existing behavior from EPD search and operational EPD list views
  
## Files changed

  - `templates/pages/building/operational_info/selected_operational_product.html` - Added conditional logic to show Generic pill for generic EPDs instead of always showing external link

baho-j committed:
Fix: generic operational EPDs showing incorrect link instead of pill
- Replace clickable link with "Generic" pill for generic operational EPDs to match behavior in EPD search.

pcschreiber1 reviewed (changes requested):
The ticket isn't limited operational carbon, it applies also to the generic EPDs in the BOQ and Component editor

baho-j committed:
Merge branch 'main' into 250-bug-link-of-operational-epds-leads-back-to-main-site

baho-j committed:
Revert "Merge branch 'main' into 250-bug-link-of-operational-epds-leads-back-to-main-site"

This reverts commit e5d0a6c7b7e197a231c85f91cacfb07413ee6959, reversing
changes made to d757dcf8e223f2afb097575898d13c7a71a0313a.

baho-j committed:
Fix: Replace broken link with Generic pill for generic EPDs in BOQ and Component editor
- Extended the fix from issue #250 to cover all areas where generic EPDs are displayed in both the Bill of Quantities and Component editor views.

pcschreiber1 reviewed (changes requested):
@Josy-coder when I open an existing component there is neither the link nor the pill

<img width="1385" height="991" alt="image" src="https://github.com/user-attachments/assets/d62ddec0-1b26-41c6-a7d8-d5dbab04c30c" />

baho-j committed:
Fix: added a missing type field in selectedepd dataclass

baho-j committed:
Merge branch 'main' into 250-bug-link-of-operational-epds-leads-back-to-main-site
